app/src/js/utils/table-config/executions.js

```diff
@@ -36,6 +37,7 @@ export const tableColumns = [
   },
   {
     Header: strings.collection_name,
-    accessor: 'collectionId'
+    accessor: 'collectionId',
+    Cell: ({ row }) => formatCollectionId(row.original.collectionId)
```

Comment:
Ah, thanks.   My react/redux is more than a little rusty.  I'll update this, however it appears that changes to this were merged since I made this update.

Reply:
Yeah, that does indeed work.   Thanks for pointing that out!